[bug] Layerwise mode retrive error

**Describe the bug**
when use layerwise transfer will cause error ,then will not storing any kvcache in remote storage
I found it may cause by incorrect shape in layerwise


```
(ServeReplica:llm:vllm_serve pid=4073258) [2025-07-25 16:08:56,486] LMCache INFO: kv_bytes <memory at 0x751e5c22d540> torch.Size([128, 2, 1024]) torch.bfloat16 MemoryFormat.KV_T2D (redis_connector.py:121:lmcache.v1.storage_backend.connector.redis_connector)
(ServeReplica:llm:vllm_serve pid=4073258) Traceback (most recent call last):
(ServeReplica:llm:vllm_serve pid=4073258)   File "/data/test_dir/lmcache/v1/storage_backend/connector/redis_connector.py", line 124, in put
(ServeReplica:llm:vllm_serve pid=4073258)     ).serialize()
(ServeReplica:llm:vllm_serve pid=4073258)       ^^^^^^^^^^^
(ServeReplica:llm:vllm_serve pid=4073258)   File "/data/test_dir/lmcache/v1/protocol.py", line 94, in serialize
(ServeReplica:llm:vllm_serve pid=4073258)     assert len(self.shape) == 4, "Shape dimension should be 4"
(ServeReplica:llm:vllm_serve pid=4073258) AssertionError: Shape dimension should be 4
(ServeReplica:llm:vllm_serve pid=4073258) [2025-07-25 16:08:56,486] LMCache ERROR: Error in put: Shape dimension should be 4 (redis_connector.py:132:lmcache.v1.storage_backend.connector.redis_connector)
```

if not use layerwise it is working well

```
(ServeReplica:llm:vllm_serve pid=4076482) [2025-07-25 16:17:19,506] LMCache INFO: kv_bytes <memory at 0x7b57e80d8640> torch.Size([2, 64, 25, 1024]) torch.bfloat16 MemoryFormat.KV_2LTD (redis_connector.py:121:lmcache.v1.storage_backend.connector.redis_connector)
```

**version**
vllm=0.9.1
lmcache=0.3.2

**lmcache config**
```
"LMCACHE_CHUNK_SIZE": "128",
"LMCACHE_LOCAL_CPU": "False",
"LMCACHE_SAVE_DECODE_CACHE": "False",
"LMCACHE_USE_EXPERIMENTAL": "True",
"LMCACHE_REMOTE_URL": "redis://172.21.30.200:6379", 
"LMCACHE_REMOTE_SERDE": "naive",
"LMCACHE_USE_LAYERWISE": "False",
```

--- a/lmcache/integration/vllm/vllm_v1_adapter.py
+++ b/lmcache/integration/vllm/vllm_v1_adapter.py
@@ -826,8 +826,7 @@ class LMCacheConnectorV1Impl:
                     continue
 
                 token_ids = request.token_ids
-                assert isinstance(token_ids, torch.Tensor)
-                assert token_ids.is_cpu
+                assert isinstance(token_ids, list)
 
                 slot_mapping = request.slot_mapping
                 assert isinstance(slot_mapping, torch.Tensor)
@@ -850,7 +849,7 @@ class LMCacheConnectorV1Impl:
                         * self._lmcache_chunk_size
                     )
 
-                store_mask = torch.ones_like(token_ids, dtype=torch.bool)
+                store_mask = torch.ones(len(token_ids), dtype=torch.bool)
                 store_mask[:skip_leading_tokens] = False
 
                 logger.info(

